index.html

```diff
@@ -214,22 +214,42 @@
                         </dl>
                     </div>
 
-                    <div class="floe-column floe-front-news">
-                        <h3>PhET Accessible Physics Simulations</h3>
-                        <p class="floe-date">Nov. 12, 2015</p>
-                        <p>Floe is working with <a href="http://phet.colorado.edu/">PhET</a>
-                            to improve the accessibility of their free, open, interactive physics simulations. Over the past
-                            year, we have contributed research, ideas, and evaluations for improvements to two of PhET's simulations.</br>
-                            <a href="./news/2015-11-16-phet.html">Read more</a></p>
+                    <div class="floe-column">
+                        <div class="floe-recent-events">
+                            <h3>Recent Events</h3>
+                            <h4>DEEP</h4>
+                            <p>On Oct. 14-15, 2015, the Inclusive Design Research Centre (IDRC)
```

Comment:
Instead of "Oct." perhaps say "October" to be consistent with "February" in the following paragraph.